src/collection.js

```diff
@@ -104,8 +106,14 @@ export default class Collection {
     });
   }
 
-  use(transfomer) {
-    this._transformers[transfomer.type].push(transfomer);
+  use(plugin) {
+    if (plugin.type == "idschema") {
+      this._idSchema = plugin;
+    } else if (plugin.type == "remotetransformer") {
+      this._transformers.remote.push(plugin);
```

Comment:
Actually, now that we have `Collection#configure(config)`, maybe we should just store that exact object as `this._config`? Then we have `this._config.idSchema` and `this._config.remoteTransformers`. Or if you want to stick to 'plugins then we can call the configure method `Collection#configurePlugins(plugins)`.